src/db_interactions.py

```diff
@@ -0,0 +1,11 @@
+import mysql.connector  # type: ignore
+
+connection = mysql.connector.connect(user="root", password="root", host="mysql", port="3306", database="db")
+print("Succesfully connected to db")
```

Comment:
Can we setup logger?

Reply:
will set up logger in next pr with actual DB implementations